### DISCUSSION TITLE
Cant use skeleton component with latest daisyUI framework

### DISCUSSION THREAD

**Discussion by altafsamnani** [saadeghi/daisyui#2599] [Q&A] [answered]
First of all thank you for making the daisy framework, somehow i am trying to use skeleton component, but skeleton class is not loaded, can you plz guide?
I am using "daisyui": "^4.4.14" and i have all right configuration in place for tailwind.config.js

**Comment by saadeghi** [comment]
The library has no issues with `skeleton` class and it's working in this example as you can see: https://play.tailwindcss.com/ft4ZWMzqyw

It's hard to guess what's the issue without seeing the code.  
Can you please provide an example code on Tailwind Play or a public GitHub repo?

**Comment by saadeghi** [comment]
> fallback vars may be ?

No. that's not the issue. that variable only must work on old browsers.
The style for skeleton looks okay in the video.

It is hard to guess what's the issu if you don't provide an example code on Tailwind Play or a public GitHub repo.

- What's the `skeleton` animation on devtools?
- What's the value of `--b1` and `--b3` variables on your theme?
- Do you have this issue on other browsers as well?
- Do you have this issue on a private thab?
- Do you have any other CSS file in your site that may change the styles on the page.

**Reply by altafsamnani** [reply]
I cant find --b1 and --b3 variables anywhere

**Reply by altafsamnani** [reply, marked as answer]
damn i restarted vite and it worked, i didnt restart vite after doing npm install 
Damn, thank you for your help
